### ISSUE TITLE
draftman lacks an import capability

### ISSUE THREAD

**Issue by johnblommers** [jsseidel/draftmangtk#9]
Once the first draft of a manuscript is complete, a writer will often compile a standalone edition and pass it around for editing. They may pass the manuscript over a grammar and spelling checker. Armed with feedback and edits, it would be useful to be able to import the edited Markdown manuscript back into draftman to take advantage of its ability to rearrange material easily, a huge strength.
Consider adding an import capability that breaks up the imported Markdown manuscript into individual files and folders containing chapters and scenes based on the Markdown headings.

**Comment by jsseidel** [comment]
Agreed. I always imagined that once a first draft was out of draftman and being edited via a word doc or whatever, that would be the end of using draftman in the process, but you may have a point.

In fact, I created something similar for an unrelated project that I may be able to retrofit here based on this: https://github.com/att/MkTechDocs/issues/29. This is the utility: https://github.com/att/MkTechDocs/blob/master/bin/markdown-2-mktechdocs.py

Granted, it's written in python, but maybe I'll see about adding this as a feature when I get a chance.
